lib/compress/zstd_compress_literals.h

```diff
@@ -26,4 +26,15 @@ size_t ZSTD_compressLiterals (ZSTD_hufCTables_t const* prevHuf,
                               void* entropyWorkspace, size_t entropyWorkspaceSize,
                         const int bmi2);
 
+/* Incorporate fast check(s) on the suspicion that this data is not compressible and back off
+   quickly to raw format if so. */
+size_t ZSTD_compressLiterals_fastCheck (ZSTD_hufCTables_t const* prevHuf,
```

Comment:
I don't think we need another function with & without `fastCheck`. This function is internal to zstd, so we can just add this parameter to `ZSTD_compressLiterals()`.